src/Elastic.Markdown/Assets/pages-nav.ts

```diff
@@ -16,11 +16,14 @@ function scrollCurrentNaviItemIntoView(nav: HTMLElement, delay: number) {
 	expandAllParents(currentNavItem);
 	setTimeout(() => {
```

Comment:
We should treat instances of `setTimeout` as a codesmell IMO. 

I am getting inconsistent behaviour refreshing the page after scrolling up but leaving the anchor at the top of the history queue.

Reply:
https://github.com/elastic/docs-builder/pull/702/commits/cec5b55c42b23889cb7195262f763a31c8721bf1

In this case, the delay was just artificial, it's not actually needed to get the right timing. I just removed it.